lib/types/channels.d.ts

```diff
@@ -1508,3 +1510,19 @@ export interface RawModalLabel extends Omit<BaseComponent, "id"> {
     label: string;
     type: ComponentTypes.LABEL;
 }
+
+export interface FileUploadComponent extends BaseComponent {
```

Comment:
should this be named ModalFileUploadComponent since it's only in modals?

Reply:
Yeah make it specific, if we get more in  the future and the structure is the same I'd rather have to make it more generic than more specific